src/dirs.rs

```diff
@@ -43,6 +46,7 @@ pub enum Dirs {
         ),
         windows
     ))]
+    /// Set of directories to specifically store the cache, config app data etc.
```

Comment:
Doc comments should precede the attributes (like for the other variant you did). That's the convention we have.